modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/web/internal/portlet/action/test/ExportImportObjectDefinitionTest.java

```diff
@@ -194,27 +243,33 @@ private ThemeDisplay _getThemeDisplay() throws Exception {
 		return themeDisplay;
 	}
 
-	private ObjectDefinition _importObjectDefinition() throws Exception {
+	private ObjectDefinition _importCustomObjectDefinition(
+			String fileName, String name,
+			ObjectDefinitionResource objectDefinitionResource)
+		throws Exception {
+
 		_mvcActionCommand.processAction(
-			_createMockLiferayPortletActionRequest(
-				"object_definition.json", "ImportedObjectDefinition"),
+			_createMockLiferayPortletActionRequest("", fileName, name),
```

Comment:
@pedro-oliveira446 use `StringPool.BLANK` instead of `""`, just to follow the pattern.

Thanks!